src/docs/sdk/event-payloads/contexts.mdx

```diff
@@ -530,3 +529,61 @@ The following example illustrates the contexts part of the <Link to="/sdk/event-
   }
 }
 ```
+
+## Cloud Context
```

Comment:
Can we add a link to https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/semantic_conventions/cloud.md?

Reply:
yes, will add (but to the documentation and not github: https://opentelemetry.io/docs/reference/specification/resource/semantic_conventions/cloud/)